test/integration/test_language_container_deployer.py

```diff
@@ -30,6 +30,7 @@ def create_container_deployer(language_alias: str,
 
 
 def test_language_container_deployer(
+        itde,
```

Comment:
The itde fixture returns a TestConfig which contains the ExasolConfig and the BucketFSConfig